qt/python/mantidqt/_common.sip

```diff
@@ -198,6 +203,12 @@ public:
       const QString &optionalMsg = QString(),
       const QStringList &enabled = QStringList(),
       const QStringList &disabled = QStringList());
+  void showHelpPage(const QString &url = QString());
```

Comment:
Are these functions used? `showCustomInterfaceHelp` seems used from the old `pymantidplot` module so I'm not sure we need these yet?

Reply:
Not used but I exposed this while attempting to get the help buttons working in workbench.
I can remove this from this PR

Reply:
:+1: